### DISCUSSION TITLE
Bluetooth Support with the release of Plasma 6

### DISCUSSION THREAD

**Discussion by SnoutBug** [andyholmes/gnome-shell-extension-gsconnect#1751] [Ideas]
### Describe your request

With the release of Plasma 6, Bluetooth was announced to be officially supported in KDE Connect. As far as I can tell changes have been made to the android app which enable bluetooth support, though a build has not yet been published.

### Proposed solution

With bluetooth being a mainline feature now, a major hurdle for contributers and testers is removed. Maybe #17 will pick up some steam again.

### GSConnect version

latest

### Installed from

GNOME Extensions website

**Comment by andyholmes** [comment]
Moving to discussions, as a duplicate of #17.

Sorry, but the issue of Bluetooth support attracts way too many "me too!" comments for the issue tracker. When someone is ready to take on the task, they are welcome to ping a maintainer from the discussion and we can re-open the original issue.

Thanks for your understanding!
